pkg/e2e/setup.go

```diff
@@ -71,11 +76,88 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// only needs to run once
 })
 
-// Collect logs after each test
-var _ = ginkgo.AfterSuite(func() {
+// After all tests run, clean up within the Ginkgo context
+var _ = ginkgo.SynchronizedAfterSuite(func() {
+	h := helper.NewOutsideGinkgo()
+	// Run Must Gather
+	func() {
+		defer ginkgo.GinkgoRecover()
+
+		log.Print("Running Must Gather...")
+		mustGatherTimeoutInSeconds := 900
+		h.SetServiceAccount("system:serviceaccount:%s:cluster-admin")
+		r := h.Runner(fmt.Sprintf("oc adm must-gather --dest-dir=%v", runner.DefaultRunner.OutputDir))
+		r.Name = "must-gather"
+		r.Tarball = true
+		stopCh := make(chan struct{})
+		err := r.Run(mustGatherTimeoutInSeconds, stopCh)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
Rather than use Expect(), let's just do manual err checks and that way Id on't think we'll need to use ginkgo.GinkgoRecover() here at all. Should also make the error messages clearer/less alarming.

Reply:
That makes sense.